great-idea-website/css/index.css

```diff
@@ -63,4 +63,94 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+.navigation {
+	display: inline-block;
+	padding: 10px 20px 20px 40px;
+	font-size: 16pt;
+}
+
+.logo {
+	margin-left: 15%;
+	top: 18px;
+	right: 20px;
+	position: absolute;
+}
+
+header {
+	display: inline-block;
+    width: 100%;
+	padding: 18px;
+	font-family: bangers, cursive;
+	font-size: 18pt;
+
+}
+
+.circle-img {
+	height: 200%;
+	width: 25%;
+	margin-left: 80%;
```

Comment:
the margin-left is causing your image to be pushed too far to the right.  play around with this property to get it like the design spec.